AmazonChimeSDKDemo/AmazonChimeSDKDemo/LiveTranscriptionOptionsViewController.swift

```diff
@@ -174,6 +377,34 @@ class LiveTranscriptionOptionsViewController: UIViewController, UITextFieldDeleg
             }
         }
     }
+    
+    @IBAction func toggleCustomLanguageTextField(_ enableCustomLangugeModelTextFieldSwitch: UISwitch) {
+        if enableCustomLangugeModelTextFieldSwitch.isOn {
+            customLanguageModelTextField.isHidden = false
+        } else {
+            customLanguageModelTextField.isHidden = true
+        }
+    }
```

Comment:
`customLanguageModelTextField.isHidden = enableCustomLangugeModelTextFieldSwitch.isOn ? false : true` ?

Reply:
will be updated.